lms/envs/aws.py

```diff
@@ -1004,6 +1005,10 @@
     'ENTERPRISE_SPECIFIC_BRANDED_WELCOME_TEMPLATE',
     ENTERPRISE_SPECIFIC_BRANDED_WELCOME_TEMPLATE
 )
+ENTERPRISE_TAGLINE = ENV_TOKENS.get(
```

Comment:
Nit: I don't think this needs to overflow a single line.

Reply:
I actually only did this to match the style (which I personally like :P ) of the surrounding assignments, including `BASE_COOKIE_DOMAIN` below, which also could be a single line but isn't.

Reply:
Gotcha. Works for me, then! I definitely prefer that style when the statement won't fit on a single line; and if that's the style elsewhere, I'm fine with it here - hence the nit + approval. ;)